src/editor/CodeHintList.js

```diff
@@ -306,14 +306,17 @@ define(function (require, exports, module) {
     };
 
     /**
-     * Check whether keyCode is one of the keys that we handle or not.
+     * Check whether Event is one of the keys that we handle or not.
      *
-     * @param {number} keyCode
+     * @param {KeyBoardEvent} keyEvent
      */
-    CodeHintList.prototype.isHandlingKeyCode = function (keyCode) {
+    CodeHintList.prototype.isHandlingEvent = function (event) {
```

Comment:
This is potentially a breaking change. One way would be deprecating the `isHandlingKeyCode` method, [throwing a deprecation warning](https://github.com/adobe/brackets/blob/master/src/utils/DeprecationWarning.js), constructing a new event with the keycode and passing it to the `isHandlingEvent`-method.

Another (simpler) way would be keeping the `isHandlingKeyCode` and checking the type, such as 

```
CodeHintList.prototype.isHandlingKeyCode = function (keyCodeOrEvent) {
  var keyCode = typeof keyCodeOrEvent === "object" ? keyCodeOrEvent.keyCode || keyCodeOrEvent;
}
```